app/src/main/res/layout/cell_view.xml

```diff
@@ -20,6 +21,7 @@
         android:id="@+id/duration_text_view"
         android:layout_width="275dp"
         android:layout_height="20dp"
+        tools:text="View 2"
```

Comment:
Is this the most useful text for you when previewing the layout? Generally would expect the value set here to be a value that this label might have when the app is in use, to make the preview as realistic as possible